src/feat/pitch-functions.cc

```diff
@@ -856,14 +857,21 @@ void OnlinePitchFeatureImpl::ExtractFrame(
   } else {
     // frame is partly in the remainder and partly in the new part.
     int32 remainder_offset = downsampled_signal_remainder_.Dim() + offset;
-    KALDI_ASSERT(remainder_offset >= 0);  // or we didn't keep enough remainder.
+    /*KALDI_ASSERT(remainder_offset >= 0);*/  // or we didn't keep enough remainder.
     KALDI_ASSERT(offset + full_frame_length > 0);  // or we should have
                                                    // processed this frame last
                                                    // time.
 
     int32 old_length = -offset, new_length = offset + full_frame_length;
-    window->Range(0, old_length).CopyFromVec(
-        downsampled_signal_remainder_.Range(remainder_offset, old_length));
+
+    // BP: this looks to me like something has not been done properly
```

Comment:
BP: I assume you are commenting on the code that has been added just below. 
The code needs to be fixed instead of just commenting that it's not right.
I do agree that it hasn't been properly done-- the part at the beginning of the file needs to be copied if it's partly there.